modules/@ergonode/products-batch-actions/src/components/Modals/UpdateProductsModal.vue

```diff
@@ -104,8 +128,8 @@ export default {
         verticalTabs() {
```

Comment:
here should probably be a space under the `verticaTabs` extension. as in other places with `verticalTabs`